src/data/sum/basic.lean

```diff
@@ -286,14 +286,16 @@ instance [decidable_rel r] [decidable_rel s] : decidable_rel (lex r s)
 protected lemma lift_rel.lex {a b : α ⊕ β} (h : lift_rel r s a b) : lex r s a b :=
 by { cases h, exacts [lex.inl ‹_›, lex.inr ‹_›] }
 
+lemma lift_rel_le_lex : subrelation (lift_rel r s) (lex r s) := λ a b, lift_rel.lex
```

Comment:
```suggestion
lemma lift_rel_subrelation_lex : subrelation (lift_rel r s) (lex r s) := λ a b, lift_rel.lex
```
or
```suggestion
lemma lift_rel_le_lex : lift_rel r s ≤ lex r s := λ a b, lift_rel.lex
```

Reply:
The latter doesn't work, since it depends on the `has_le` instance for `Prop`.